src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

```diff
@@ -79,7 +79,6 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsLoggingAbstractionsVersion)" />
-    <PackageReference Include="Microsoft.Identity.Abstractions" Version="$(MicrosoftIdentityAbstractionsVersion)" />
```

Comment:
We should also remove this, since it's not used:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/1badce8949d974cf2650fa496432fadd0e3aeee8/build/dependencies.props#L6